### PR TITLE
Avoid folly::init overwirte log configuration.

### PR DESCRIPTION
**Pull request by Shylock-Hg**
## What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### Description:
Avoid folly::init overwirte the glog configuration.

## How do you solve it?
Patch of #4409 

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

File: src/daemons/MetaDaemon.cpp [hunk 1]
```diff
@@ -66,12 +66,7 @@ int main(int argc, char* argv[]) {
   // the 2nd will make the 1st failed to output log anymore
```
Comment (critical27) [inline src/daemons/MetaDaemon.cpp:66]: Please verify if the behavior in the comment.

Comment (Shylock-Hg) [reply src/daemons/MetaDaemon.cpp:66]: In current `folly::init` and `SetUpLogging` is called after pid checking too.

Comment (Shylock-Hg) [reply src/daemons/MetaDaemon.cpp:66]: But Standalone daemon don't require this constraint even before this PR, need to modify it?

**Review by nevermore3 — APPROVED** [review]

File: src/daemons/MetaDaemon.cpp (hunk 1 above)
Comment (critical27) [reply src/daemons/MetaDaemon.cpp:66]: It is ok for now. I can't clearly remember why we move `setupLogging` very close to start position of the main.

**Review by critical27 — APPROVED** [review]

**Pull request merged**